.github/workflows/cypress-run.yml

```diff
@@ -0,0 +1,32 @@
+name: cypress-run
+on: push
+jobs:
+  cypress-run:
+      runs-on: ubuntu-latest
+      # Browser on which the tests are run
+      container: cypress/browsers:node12.18.3-chrome87-ff82
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+        - name: Setup node
+          uses: actions/setup-node@v1
+          with:
+            node-version: '12.x'
+        - name: Install dependencies
+          run: yarn
+        - name: Browser tests
+          uses: cypress-io/github-action@v2
+          with:
+            # Your starting script
+            start: yarn start:test-headless
+        # Creates and uploads GitHub artifacts in case of failure
+        - uses: actions/upload-artifact@v1
+          if: failure()
+          with:
+            name: cypress-screenshots
+            path: cypress/screenshots
+        - uses: actions/upload-artifact@v1
+          if: failure()
+          with:
+            name: cypress-videos
+            path: cypress/videos
```

Comment:
Please add a new line at the end of files.

To avoid this kind of error, you can download the editorconfig plugin, depending on your text editor. After downloading it, every time you will save your file, it will automatically apply the basic style rules to your file (removing trailing whitespaces, adding a new line at the end of file...) thanks to the .editorconfig file in this repo.
See more about editorconfig: https://editorconfig.org/#overview
And the editorconfig file of this repo: https://github.com/meilisearch/docs-searchbar.js/blob/main/.editorconfig